### PR TITLE
Fix: Error document

### PR DESCRIPTION
**Pull request by Kininaru**
Fix: https://github.com/casbin/casbin-website/issues/174

### CODE REVIEW

File: docs/WebpageController.md [hunk 1]
```diff
@@ -30,15 +30,39 @@ const permission = {
...
+});
+// output: you can write data1
 ```
 
-You can refer to our [React example](https://github.com/casbin-js/examples) to see a practical usage of Casbin.js
+and `cannot()` is used in the same way:
+
+```javascript
+result = authorizer.cannot("read", "data2");
+result.then((success, failed) => {
```
Comment (Zxilly) [inline docs/WebpageController.md:53]: same here

File: docs/WebpageController.md [hunk 2]
```diff
@@ -72,9 +96,12 @@ const authorizer = new casbinjs.Authorizer(
 authorizer.setUser("Tom");
 
 // Evaluate the permission
-if authorizer.can("read", "data1") {
-    // Some frontend procedure ...
-}
+result = authorizer.can("read", "data1");
+result.then((success, failed) => {
```
Comment (Zxilly) [inline docs/WebpageController.md:100]: same here

File: docs/WebpageController.md [hunk 3]
```diff
@@ -30,15 +30,39 @@ const permission = {
...
+```
 
-authorizer.setPermission(permission);
-console.log(authorizer.can("read", "data1"));  // True
-console.log(authorizer.cannot("write", "data2"));  // True
+now we got an authorizer `authorizer`. We can get permission rules from it by using the API `authorizer.can()` and `authorizer.cannot()`. The return values of these 2 APIs are JavaScript Promises ([details here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises)), so we should use the `then()` method of the return value like this:
+
+```javascript
+result = authorizer.can("write", "data1");
+result.then((success, failed) => {
```
Comment (Zxilly) [inline docs/WebpageController.md:39]: There is no need for `failed`, enforcer will never reject

**Review by Zxilly — CHANGES_REQUESTED** [review]

File: docs/WebpageController.md (hunk 3 above)
Comment (Kininaru) [reply docs/WebpageController.md:39]: The electricity in our dormitory has been cut off. I will fix that tomorrow.

**Review by Zxilly — APPROVED** [review]

**Pull request merged**